Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -200,7 +200,7 @@ static inline uint8_t authDataFlags(ClientDataType type, LocalConnection::UserVe
 {
     Vector<WebCore::AuthenticatorTransport> transports = { WebCore::AuthenticatorTransport::Internal };
     if (shouldUpdateQuery())
-        transports.append(WebCore::AuthenticatorTransport::Cable);
+        transports.append(WebCore::AuthenticatorTransport::Hybrid);
```

Comment:
I guess this is the only part where we actually use the 'rename'. It seems like you are keeping the old 'Cable' transport around for backwards compatibility? Or will it get removed once we confirm no regressions from this change?

Reply:
Yes, I don't want to type error if an RP gives it in allowCredentials from old Chrome/Safari.

 It turns out[ all three browsers ](https://github.com/w3c/webauthn/issues/1738) ignored the specification that unknown enum types should be ignored. We can remove AuthenticatorTransport::Cable in https://bugs.webkit.org/show_bug.cgi?id=241517 . I added a comment to remind me.